docs/03-tutorials/00-eventing/evnt-03-introduction-to-jetstream.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Introduction to JetStream
+---
+
+JetStream is the 'built-in distributed persistence system' of NATS. You can read more about it [here](../../05-technical-reference/00-architecture/evnt-01-architecture.md#jetstream).
+
+## Streams and Consumers
+
+A `Stream` is a 'message store' for the published events. In Kyma, we use only one stream for all the events. You can configure the retention and delivery policies for the stream, based on the use-case.
```

Comment:
```suggestion
A Stream specifies how messages are stored for the published events. In Kyma, we use only one stream for all the events. You can configure the retention and delivery policies for the stream, depending on the use case.
```

Reply:
It doesn't specify how it stores. It basically is the store.

Reply:
Ok, how about now?

Reply:
This is better!